trp_use_case/view/use_case_collection.xml

```diff
@@ -12,7 +12,8 @@
                             <field name="partner_id"/>
                             <field name="create_date" readonly="True" />
                             <field name="create_uid" readonly="True" />
-                        </group>
+			    <field name="x_status"/>
```

Comment:
by convention, the status fields goes into a `header` tag at the top and uses the statusbar widget. Use this opportunity to wrap everything into a `sheet` tag